bigbluebutton-html5/private/config/settings.yml

```
@@ -768,6 +768,7 @@ public:
     # e.g.: disableEmojis: ['grin','laughing']
     disableEmojis: []
     allowedElements: ['a', 'code', 'em', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'li', 'ol', 'ul', 'p', 'strong']
+    toolbar: ['replies']
```

```suggestion
    # available options for toolbar: reply
    toolbar: []
```

I think it should come disabled by default (for a while).